src/services/transactionHistory.ts

```diff
@@ -263,6 +304,17 @@ export const askTransactionHistory = async (
             assets: extractAssets(obj.f5)
       }))
       : []; 
+    const collaterals = row.collateralInAddrValPairs 
+      ? row.collateralInAddrValPairs.map((obj: any): TransInputFrag => ({
+            address: obj.f1,
+            amount: obj.f2.toString(),
+            id: obj.f3.concat(obj.f4.toString()),
+            index: obj.f4,
+            txHash: obj.f3,
+            assets: extractAssets(obj.f5)
+      }))
+      : []; 
+    inputs = inputs.concat(collaterals);
```

Comment:
@pedromtcosta , collateral inputs must be added to each tx as a new separate field
```
inputs: ... ,
collateralInputs: ... ,
outputs: ... ,
```

Reply:
Fixed!